docs/intro.md

```diff
@@ -5,7 +5,9 @@ sidebar_label: Overview
 ---
 # Overview of Komiser Documentation
 
-![Tailwarden](/img/docs-banner-tw.png)
+
+![Tailwarden](/img/komiser-overview-light.png#gh-light-mode-only)![Tailwarden](/img/komiser-overview-dark.png#gh-dark-mode-only)
+
 [![Twitter URL](https://img.shields.io/twitter/url/https/twitter.com/fold_left.svg?style=social&label=Follow%20%40mlabouardy)](https://twitter.com/mlabouardy) [![Tweet](https://img.shields.io/twitter/url/http/shields.io.svg?style=social)](https://twitter.com/intent/tweet?text=Optimize%20Cost%20and%20Security%20on%20AWS&url=https://github.com/mlabouardy/komiser&via=mlabouardy&hashtags=komiser,aws,gcp,cloud,serverless,devops)
```

Comment:
can you change from my twitter account to tailwarden account

Reply:
Done